src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -24,15 +25,16 @@ const useSaveAnonChatHook = () => {
     return channelListSchema;
   };
 
-  const helperGoToAnonymousChat = async (object: InitAnonymousChatData) => {
+  const helperGoToAnonymousChat = async (object: InitAnonymousChatData, from: string) => {
     const channelListSchema = await helperFindChatById(object);
-    goToChatScreen(channelListSchema);
+    goToChatScreen(channelListSchema, from);
   };
 
   const saveChatFromOtherProfile = async (
     object: InitAnonymousChatData,
     status = 'sent',
-    withNavigate = false
+    withNavigate,
+    type: string
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `saveChatFromOtherProfile` now accepts a new parameter `type: string`. Ensure that all calls to this function are updated to include this new argument.

```diff
- const saveChatFromOtherProfile = async (
-   object: InitAnonymousChatData,
-   status = 'sent',
-   withNavigate = false
- ) => {
+ const saveChatFromOtherProfile = async (
+   object: InitAnonymousChatData,
+   status = 'sent',
+   withNavigate,
+   type: string
+ ) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->